oxd-server/src/test/java/org/gluu/oxd/mock/listener/EnableMocksListener.java

```diff
@@ -0,0 +1,35 @@
+package org.gluu.oxd.mock.listener;
+
+import org.testng.IAnnotationTransformer;
+import org.testng.ITest;
+import org.testng.annotations.ITestAnnotation;
+import org.testng.annotations.Parameters;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+
+public class EnableMocksListener implements IAnnotationTransformer {
```

Comment:
I guess we can add "mock" package exclusion since load test will be run only manually.

Reply:
Done